data/avgi/avgi outfits.txt

```diff
@@ -433,7 +432,6 @@ outfit "Inductive Extractor"
 	thumbnail "outfit/inductive extractor"
 	"mass" 12
 	"outfit space" -12
-	"engine capacity" -12
```

Comment:
If you're going to remove the engine capacity cost from this, you'd need to tone down the energy it generates considerably to keep it balanced. The current stats were only balanced because of the high engine space cost.